etc/terraform/main.tf

```diff
@@ -0,0 +1,133 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  profile = "private"
```

Comment:
This will only work on your established profile, we might add info that this is purely for local/own instance testing. You might want to check what we have in here: https://github.com/CodeForPoznan/infrastructure, that's the whole organisation's infrastructure repository.